master/buildbot/status/web/jsontestresults.py

```diff
@@ -93,3 +95,6 @@ def content(self, req, cxt):
 
         template = req.site.buildbot_service.templates.get_template("jsontestresults.html")
         return template.render(**cxt)
+
+def removeTestFilter(s):
+    return re.sub('[-]{1,2}testfilter=.*\s', '', s)
```

Comment:
Would be great if you check if the 's' parameter is not 'None' or you expecting exception in this case?